uuid/uuid.go

```diff
@@ -27,6 +27,21 @@ func IsAndroid(uuid string) bool {
 	return IsValidFast(uuid) && matchUuidRegexAndroid(uuid) && !isTest(uuid)
 }
 
+type IdentityType int
+
+const (
+	IdentityTypeOther IdentityType = iota
+	IdentityTypeIDFAOrAAID
+	IdentityTypeNoid
+)
+
+func GetIdentity(uuid string) IdentityType {
+	if IsNoid(uuid) {
```

Comment:
I would prefer if this function is renamed `IsNoID` so you see it is a separate word like in pivot.

Reply:
sure, if it is easier to read lets do so